moped-editor/src/views/projects/projectView/ProjectStatusBadge.js

```diff
@@ -206,7 +201,8 @@ const ProjectStatusBadge = ({
         condensed ? chipClasses.condensed : chipClasses.root,
         leftMargin && chipClasses.leftMargin
       )}
-      icon={<ChipIcon className={iconClasses.root} />}
+      // for some reason passing color as a prop is ensuring the iconClasses.root style is being used
+      icon={<ChipIcon className={iconClasses.root} color="action" />}
```

Comment:
this is the hackiest part of the whole upgrade -- the iconClasses.root style is ignored unless I pass the color prop in, then the color gets overwritten by the class style.